core/lib/libtomcrypt/ecc.c

```diff
@@ -393,6 +393,30 @@ static const struct crypto_ecc_public_ops sm2_kep_public_key_ops = {
 	.free = _ltc_ecc_free_public_key,
 };
 
+const struct crypto_ecc_keypair_ops *
```

Comment:
I'd like commit "libtomcrypt/mbedtls: ecc: support the crypto driver" to be split in 3.